sklearn/semi_supervised/self_training.py

```diff
@@ -41,7 +41,9 @@ class SelfTrainingClassifier(BaseEstimator):
 
     max_iter : integer, optional (default=20)
         Maximum number of iterations allowed. Should be greater than or equal
-        to 0.
+        to 0. If it is None, the classifier will continue to predict labels
```

Comment:
Use double backticks for None: \`\`None\`\`

Also "Note that if it is None" -> "In this case..."